Libraries/src/Amazon.Lambda.RuntimeSupport/Helpers/ConsoleLoggerWriter.cs

```diff
@@ -107,7 +128,7 @@ public class LogLevelLoggerWriter : IConsoleLoggerWriter
         /// Amazon.Lambda.Core can not be relied on because the Lambda Function could be using
         /// an older version of Amazon.Lambda.Core before LogLevel existed in Amazon.Lambda.Core.
         /// </summary>
-        enum LogLevel
+        public enum LogLevel
```

Comment:
This doesn't need to be public. Users interact with the `LogLevel` enum from Amazon.Lambda.Core. RuntimeSupport has to have a mirror of this enum because it can't rely on having the latest copy of Amazon.Lambda.Core when running that would have the enum.